electrum/gui/kivy/uix/dialogs/cpfp_dialog.py

```diff
@@ -0,0 +1,161 @@
+from typing import TYPE_CHECKING, Optional
+
+from kivy.app import App
+from kivy.factory import Factory
+from kivy.properties import ObjectProperty
+from kivy.lang import Builder
+
+from electrum.gui.kivy.i18n import _
+
+if TYPE_CHECKING:
+    from ...main_window import ElectrumWindow
+
+from .fee_dialog import FeeSliderDialog
+from electrum.gui.qt.amountedit import BTCAmountEdit
+from electrum.util import format_satoshis_plain
+
+Builder.load_string('''
+<CPFPDialog@Popup>
+    title: _('CPFP')
+    size_hint: 0.8, 0.8
+    pos_hint: {'top':0.9}
+    method: 0
+    BoxLayout:
+        orientation: 'vertical'
+        padding: '10dp'
+        GridLayout:
+            height: self.minimum_height
+            size_hint_y: None
+            cols: 1
+            spacing: '10dp'
+            BoxLabel:
+                id: msg1
+                text: _('CPFP stands for Child Pays for Parent')
```

Comment:
You should use e.g. `TopLabel` instead of `BoxLabel`. `BoxLabel` was made for text-value pairs, so it ~halves the available horizontal space if you use it for just text.
Also, note that `id` is unused.

```suggestion
            TopLabel:
                text:
                    _(\
                    "A CPFP is a transaction that sends an unconfirmed output back to "\
                    "yourself, with a high fee. The goal is to have miners confirm "\
                    "the parent transaction in order to get the fee attached to the "\
                    "child transaction.")
```